0000-ldap-sync-tool.md

```diff
@@ -0,0 +1,139 @@
+- Feature Name: `mgr-ldapsync`: LDAP sync tool
+- Start Date: 2019, Oct 28
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The `mgr-ldapsync` is a tool that performs synchronisation of the
```

Comment:
uni-directional or bi-directional synchronization?

Reply:
Loud thinking:

... configures and manages users in Uyuni from a directory service.

Reply:
@mcalmer copypaste that to summary section, thanks! :wink: